crates/bevy_pbr/src/material.rs

```diff
@@ -9,6 +10,8 @@ pub struct StandardMaterial {
     pub albedo: Color,
     #[shader_def]
     pub albedo_texture: Option<Handle<Texture>>,
+    /// Represented as roughness/metallic.
+    pub pbr: Vec2,
```

Comment:
I'm inclined to split the pbr: Vec2<f32> into separate roughness_factor: f32 and metallic_factor: f32, to make this more clear to the end user.

Reply:
We should also note that the input is actually perceptualRoughness, a linear parameter that's then mapped to roughness in the shader (paraphrased `float roughness = perceptualRoughness * perceptualRoughness`).

Reply:
Added some comments to `material.rs`